src/OrchardCore/OrchardCore.Apis.GraphQL.Abstractions/README.md

```diff
@@ -157,9 +157,117 @@ Shown in the example above, we have an autoroutePart argument, this is registere
 
 Done.
 
+## Querying related content items
+
+One of the features of Content Items, is that they can be related to other Content Items.
+
+Imagine we have the following Content Types: (abbreviated for readability)
```

Comment:
You don't need to provide some json, just mention types that have Content Picker fields.